### PR TITLE
added coinscouts

### PR DESCRIPTION
**Pull request by Screpuh**

### CODE REVIEW

File: data/coinscouts.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+```
+name:CoinScouts 
+description:CoinScouts is the world's most powerful crypto database. It's a community-driven one-stop platform for finding, analyzing, and discussing cryptocurrencies. Using special filters, you can DYOR with just a few clicks! 
+token_image_url:https://coinscouts.io/images/coinscouts_logo.png
+ticker:
+website_url:https://coinscouts.io
+webapp_url:
+contract_url:https://coinscouts.io/contact
```
Comment (pentcle) [inline data/coinscouts.md:8]: hello - this is for contract url (not contact) :)

File: data/coinscouts.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+```
+name:CoinScouts 
+description:CoinScouts is the world's most powerful crypto database. It's a community-driven one-stop platform for finding, analyzing, and discussing cryptocurrencies. Using special filters, you can DYOR with just a few clicks!
```
Comment (pentcle) [inline data/coinscouts.md:3]: CoinScouts is a crypto database. It's a community-driven one-stop platform for finding, analyzing, and discussing cryptocurrencies. Using their filters, you can DYOR with just a few clicks

**Review by pentcle — CHANGES_REQUESTED** [review]
hello - thanks for adding - a couple of comments - cheers again !!

File: data/coinscouts.md (hunk 2 above)
Comment (Screpuh) [reply data/coinscouts.md]: thanks for checking!

**Pull request merged**